parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -951,8 +951,8 @@ enableLocalDNS() {
     local enable_localdns_result=$?
 
     if [ "$enable_localdns_result" -ne 0 ]; then
```

Comment:
how about we just simplify this whole thing to:

```suggestion
    if ! systemctlEnableAndStart localdns 30; then
      echo "Enable localdns failed."
      return $ERR_LOCALDNS_FAIL
    fi
```

Reply:
the `return 0` at the end is also redundant

Reply:
good point